asciidoc/modules/con_mg_generic-updates-in-vertx-http.adoc

```diff
@@ -109,11 +109,13 @@ The following example shows how to set WebSocket connections in {VertX} {v4}.
 options.setMaxWebSockets(30); // Maximum connection is set to 30 for each endpoint
 ----
 
-== Added `HttpMethod` as a new interface and removed the `HttpMethod` enumeration type
+== `HttpMethod` is available as a interface
 
-`HttpMethod` is a new interface. In earlier releases of {VertX}, `HttpMethod` was declared as an enumerated data type. As an enumeration, it limited the extensibility of HTTP. Further, it prevented serving other HTTP methods with this type directly. You would have to use the  `HttpMethod.OTHER` value along with the `rawMethod` attribute during server and client HTTP requests.
+`HttpMethod` is available as a new interface.
 
-If you are using `HttpMethod` enumerated data type in a switch block, you can use the following code to migrate your application to {VertX} {v4}.
+In earlier releases of {VertX}, `HttpMethod` was declared as an enumerated data type. As an enumeration, it limited the extensibility of HTTP. Further, it prevented serving other HTTP methods with this type directly. You had to use the  `HttpMethod.OTHER` value along with the `rawMethod` attribute during server and client HTTP requests.
```

Comment:
Can you move the changes in this file to another PR?

Reply:
@tsegismont This file does not require review. I rephrased some sections. Is it okay to merge it in this PR?